change/@microsoft-teams-js-77061c12-8716-4a1d-8a14-c09b9d1041e4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Remove legacy AFR endpoints",
```

Comment:
Change file guidelines require past tense, i.e., "Removed legacy endpoints from `IFluidTenantInfo` interface." (I don't know if any teams-js developers would know what "AFR" stands for, but if so, feel free to add that before "endpoints" in the message.)

Reply:
Please note use of backticks as well